cinema_project/asgi.py

```diff
@@ -0,0 +1,16 @@
+"""
+ASGI config for cinema_project1 project.
```

Comment:
There seems to be a typo in the project name. It should be 'cinema_project' instead of 'cinema_project1'.